src/js/disability-benefits/components/ClaimEstimationPage.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Link } from 'react-router';
```

Comment:
Should the content on this page (L33-34) be changed?
https://github.com/department-of-veterans-affairs/vets-website/pull/4699/files#diff-fb9d863698fbadc576db2ebce1749658R34